### PR TITLE
672 referral schema

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#672 

- rebuild DB

### CODE REVIEW

File: client/src/components/Common/Icon/icons/Email.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    width={width}
+    height={height}
+    viewBox="0 0 30 20"
+    fill={color}
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M28.4615 0H1.5385C0.692285 0 0 0.692285 0 1.53844V18.4615C0 19.3076 0.692285 20 1.5385 20H28.4616C29.3077 20 30.0001 19.3078 30.0001 18.4615V1.53844C30 0.692285 29.3077 0 28.4615 0ZM27.8842 1.15377L15.885 10.1539C15.6693 10.3181 15.3381 10.4208 14.9999 10.4193C14.6619 10.4208 14.3307 10.3181 14.1149 10.1539L2.11576 1.15377H27.8842ZM21.4746 10.7477L28.0131 18.8247C28.0196 18.8328 28.0277 18.8389 28.0346 18.8463H1.96535C1.97227 18.8386 1.98035 18.8328 1.98691 18.8247L8.52539 10.7477C8.72572 10.5001 8.68775 10.137 8.43961 9.93615C8.19193 9.73582 7.82883 9.77379 7.62844 10.0215L1.15383 18.0196V1.87494L13.423 11.0769C13.8842 11.4203 14.4453 11.5715 14.9999 11.573C15.5537 11.5719 16.1153 11.4207 16.5768 11.0769L28.8459 1.87494V18.0195L22.3715 10.0215C22.1711 9.77385 21.8077 9.73576 21.5603 9.93615C21.3122 10.1365 21.2742 10.5001 21.4746 10.7477Z"
+      fill="#09C7E7"
```
Comment (thejoefriel) [inline client/src/components/Common/Icon/icons/Email.js:15]: this should be the color prop right?

File: client/src/components/Common/Icon/icons/Whatsapp.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    width={width}
+    height={height}
+    viewBox="0 0 30 30"
+    fill={color}
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M15.0037 0H14.9963C6.72563 0 0 6.7275 0 15C0 18.2812 1.0575 21.3225 2.85562 23.7919L0.98625 29.3644L6.75187 27.5213C9.12375 29.0925 11.9531 30 15.0037 30C23.2744 30 30 23.2706 30 15C30 6.72938 23.2744 0 15.0037 0Z"
+      fill="#09C7E7"
```
Comment (thejoefriel) [inline client/src/components/Common/Icon/icons/Whatsapp.js:15]: as above

File: client/src/components/Common/LinkCopy/index.js [hunk 3]
```diff
@@ -4,6 +4,15 @@ import styled from 'styled-components';
 import Icon from '../Icon';
 import * as T from '../Typography';
 
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  background: #fffbfd;
```
Comment (thejoefriel) [inline client/src/components/Common/LinkCopy/index.js:10]: make this theme color - add the color to theme if we don't have it in there

File: client/src/components/Pages/ReferralSchema/style.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  }
+`;
+
+export const Container = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const ShareWrapper = styled.div`
+  margin-right: 25px;
```
Comment (thejoefriel) [inline client/src/components/Pages/ReferralSchema/style.js:52]: use theme spacing

File: client/src/helpers/margins.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
+import { css } from 'styled-components';
```
Comment (thejoefriel) [inline client/src/helpers/margins.js:1]: AWESOME! Definitely good idea to have this abstracted into a helper

File: client/src/components/Common/Icon/icons/Email.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/components/Common/Icon/icons/Email.js:15]: done

File: client/src/components/Common/Icon/icons/Whatsapp.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/components/Common/Icon/icons/Whatsapp.js:15]: done

File: client/src/components/Common/LinkCopy/index.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/components/Common/LinkCopy/index.js:10]: done

File: client/src/components/Pages/ReferralSchema/style.js (hunk 4 above)
Comment (RamyAlshurafa) [reply client/src/components/Pages/ReferralSchema/style.js:52]: we don't have 25px in the theme,

**Pull request merged**